2024-06-06 04:27:39.323 [ERROR:envisage.plugin]: getting extensions from DeviceViewerPlugin(id='envisage_sample.widgets.device_viewer', name='Device Viewer'), trait <tasks> File "C:\Users\vigne\micromamba\envs\Microdrop-Next-Gen\lib\site-packages\envisage\plugin.py", line 331
Traceback (most recent call last):
  File "C:\Users\vigne\micromamba\envs\Microdrop-Next-Gen\lib\site-packages\envisage\plugin.py", line 328, in _get_extensions_from_trait
    extensions = getattr(self, trait_name)
  File "C:\Users\vigne\Documents\Microdrop-Next-Gen\envisage_sample\widgets\device_viewer\plugin.py", line 48, in _tasks_default
    from .task import DeviceViewerTask
  File "C:\Users\vigne\Documents\Microdrop-Next-Gen\envisage_sample\widgets\device_viewer\task.py", line 12, in <module>
    class DeviceViewerTask(Task):
  File "C:\Users\vigne\Documents\Microdrop-Next-Gen\envisage_sample\widgets\device_viewer\task.py", line 30, in DeviceViewerTask
    svg_files = List(File, default_value=[
  File "C:\Users\vigne\micromamba\envs\Microdrop-Next-Gen\lib\site-packages\traits\trait_types.py", line 2678, in __init__
    super().__init__(value, **metadata)
TypeError: TraitType.__init__() got multiple values for argument 'default_value'
